src/functions/clear.js

```diff
@@ -17,30 +17,17 @@ module.exports = async (d) => {
         d.aoiError.fnError(d, "custom", {}, "Failed To Fetch Messages With Reason: " + err);
     });
 
-    filters = filters
-        .split("}")
-        .map(function (x) {
-            x = x.replace("{", "").trim();
-            if (x.length === 0) return null;
-            let [key, value] = x.split(":");
-            return { [key]: value.split(",") };
-        })
-        .filter(Boolean);
-
-    messages = [...messages.values()]
-        .filter((x) => {
-            let passed = false;
-            for (let filter of filters) {
-                let key = Object.keys(filter)[0];
-                let value = filter[key];
-                if (key === "users" && value.includes(x.author?.id)) passed = true;
-                if (key === "words" && value.some((word) => x.content.includes(word))) passed = true;
-                if (key === "bots" && x.author?.bot === (value === "true")) passed = true;
-                if (key === "unpinned" && x.pinned !== (value === "true")) passed = true;
-            }
-            return passed;
-        })
-        .slice(0, amount);
+    if (!messages) return;
```

Comment:
??

Reply:
needed in case of messages not being there (for any reason)